### PR TITLE
Extract FileChooser from PhotoChooser

### PR DESCRIPTION
**Pull request by karthikbtf**
- we need a generic file upload component to upload docs
- extracted out FileChooser from PhotoChooser
- added a story

FileChooser - Default ⋅ Storybook 2024-02-27 13-05-20

### CODE REVIEW

**Comment by karthikbtf** [comment]
publish rc

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { ThemeProvider, StyledEngineProvider } from '@mui/material';
+import { MemoryRouter } from 'react-router';
```
Comment (joshbuchea) [inline .storybook/preview.js:3]: TIL what a `MemoryRouter` is 💡

**Review by joshbuchea — APPROVED** [review]

**Review by nickgraz — APPROVED** [review]
Nice!

File: .storybook/preview.js (hunk 1 above)
Comment (karthikbtf) [reply .storybook/preview.js:3]: Oh well, I may have to remove this.. for some reason the link option is not working well in terraware-web. Seeing `You should not use <Link> outside a <Router>` even if it is within a router. Probably missing something.

**Comment by karthikbtf** [comment]
publish rc

**Pull request merged**